curriculum/challenges/english/25-front-end-development/quiz-javascript-comparisons-and-conditionals/66edc47c11492ac5cf258ad9.md

```diff
@@ -17,439 +17,505 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder answer
+`===` checks both values and its data type, while `==` only checks for value.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+console.log(5 === "5");
+```
+
+What's the output of the code above?
 
 #### --distractors--
 
-Placeholder distractor 1
+`true`
 
 ---
 
-Placeholder distractor 2
+`NaN`
 
 ---
 
-Placeholder distractor 3
+An error is raised.
 
 #### --answer--
 
-Placeholder answer
+`false`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is NOT an comparison operator?
 
 #### --distractors--
 
-Placeholder distractor 1
+`===`
 
 ---
 
-Placeholder distractor 2
+`!==`
 
 ---
 
-Placeholder distractor 3
+`<=`
 
 #### --answer--
 
-Placeholder answer
+`?==`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+`||` in Javascript is also known as...
 
 #### --distractors--
 
-Placeholder distractor 1
+XOR
 
 ---
 
-Placeholder distractor 2
+AND
 
 ---
 
-Placeholder distractor 3
+NOT
 
 #### --answer--
 
-Placeholder answer
+OR
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+console.log(5 === 2 + 3 || 4 == 2);
+```
+
+What's the output of the code above?
 
 #### --distractors--
 
-Placeholder distractor 1
+`undefined`
 
 ---
 
-Placeholder distractor 2
+`false`
 
 ---
 
-Placeholder distractor 3
+An error is raised.
 
 #### --answer--
 
-Placeholder answer
+`true`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a truthy and falsy value?
 
 #### --distractors--
 
-Placeholder distractor 1
+Value that's both `true` and `false`.  
 
 ---
 
-Placeholder distractor 2
+Value that changes depending on context.
 
 ---
 
-Placeholder distractor 3
+Value that functions differently than `true` or `false` value.
 
 #### --answer--
 
-Placeholder answer
+A value that is considered `true` or `false` based on its content.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+if (1) {
+  console.log("True!");
+}
+```
+
+What's printed to the console from the code above?
 
 #### --distractors--
 
-Placeholder distractor 1
+`false`
 
 ---
 
-Placeholder distractor 2
+Nothing gets printed to the console.  
 
 ---
 
-Placeholder distractor 3
+An error is raised.  
 
 #### --answer--
 
-Placeholder answer
+`"True!"`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is NOT a truthy value?
 
 #### --distractors--
 
-Placeholder distractor 1
+`"false"`
 
 ---
 
-Placeholder distractor 2
+`13`
 
 ---
 
-Placeholder distractor 3
+`-1`
 
 #### --answer--
 
-Placeholder answer
+`0`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What's the difference between `undefined` and `null`?
 
 #### --distractors--
 
-Placeholder distractor 1
+No difference.
 
 ---
 
-Placeholder distractor 2
+`null` automatically declared. (When there's no value)
 
 ---
 
-Placeholder distractor 3
+`undefined` means a variable is deliberately empty, while `null` is declared, but unassigned.
```

Comment:
```suggestion
`null` is a global property, `undefined` is not.
```

I don't really understand what was trying to be communicated here. But I'm guessing it was to be the opposite of what was written in the previous distractor.

This is just something else we could use instead.